index.bs

```diff
@@ -5887,6 +5873,23 @@ dictionary MLOperandDescriptor {
   </div>
 </details>
 
+<details open algorithm>
+  <summary>
+    To <dfn for="MLOperandDescriptor">check dimensions</dfn> given {{MLOperandDescriptor}} |descriptor|, run the following steps:
+  </summary>
+  <div class=algorithm-steps>
+    1. If |descriptor|'s [=byte length=] is not supported by the implementation, then return false.
+    1. If |descriptor|.{{MLOperandDescriptor/dimensions}}'s [=list/size=] is too large to be supported by the implementation, return false.
+
+        Issue(456): The maximum number of operand dimensions is not defined, but native ML APIs usually have a maximum supported size.
+
+    1. Return true.
+  </div>
+
```

Comment:
extra spacing intended?

Reply:
Yeah, otherwise the Issue is not recognized and formatted correctly.